src/applications/disability-benefits/all-claims/tests/utils.unit.spec.jsx

```diff
@@ -1082,4 +1083,29 @@ describe('526 v2 depends functions', () => {
         .be.false;
     });
   });
+
+  describe('confirmationEmailFeature', () => {
+    it('should return true when form526 confirmation feature flags values of true', () => {
+      expect(
+        confirmationEmailFeature({
+          featureToggles: {
+            /* eslint-disable camelcase */
+            form526_confirmation_email: true,
+            form526_confirmation_email_show_copy: true,
+          },
+        }),
+      ).to.be.true;
+    });
+    it('should return false when either form526 confirmation feature flags values of false', () => {
+      expect(
+        confirmationEmailFeature({
+          featureToggles: {
+            /* eslint-disable camelcase */
```

Comment:
Disabled camelcase linting rule because the feature toggle names are snackcase, and we will be removing these feature toggles in the very near future.